tests/test_api.py

```diff
@@ -4,7 +4,9 @@
 import requests
 import requests_mock
 import unittest2 as unittest
+import pytest
 
+from time import sleep
```

Comment:
Is this needed?  I don't see it referenced in the diff.  Same with `pytest`

Reply:
Leftover from another attempt at triggering timeouts. Removing